### ISSUE TITLE
Missing 2GIOTAS since Snapshot

### ISSUE THREAD

**Issue by alomega546** [iotaledger/wallet#410]
I am missing 2 GIOTAS in my Wallet since the last Snapshot.

Shortly before I transefered these 2 GIOTAS from one Wallet to another. The Status was Pending. Then there was the Snapshot and all Nodes where down.

Yesterday I logged in the new Wallet and reattached all adresses in both wallets. I also regenerated the adress I was transfering the 2 GIOTAS. Zero balance in both Wallets. Any Idea what I can do?

**Comment by wonkytonky** [comment]
if it was still pending.. it should be in the old wallet;.   keep generating more  addresses.

**Comment by alomega546** [comment]
Thanks for your help. I attached all adresses that I used before. In the meanwhile way more than I ever used.

**Comment by wonkytonky** [comment]
yes  only attache  addresses   as much as you created before snapshot.. ..     eventually you should see value in the last address you created before snapshot
